third_party/terraform/website/docs/r/cloudfunctions_function.html.markdown

```diff
@@ -105,10 +105,8 @@ The following arguments are supported:
 
 * `name` - (Required) A user-defined name of the function. Function names must be unique globally.
 
-* `runtime` - (Optional) The runtime in which the function is going to run. One
-of `"nodejs6"`, `"nodejs8"`, `"nodejs10"`, `"python37"`, `"go111"`. If empty,
-defaults to `"nodejs6"`. It's recommended that you override the default, as
-`"nodejs6"` is deprecated.
+* `runtime` - (Required) The runtime in which the function is going to run. One
```

Comment:
```suggestion
* `runtime` - (Required) The runtime in which the function is going to run. Eg.
```

(also dropping the "of" on the line below- don't apply this directly)

While we probably don't want to validate the list of languages since we expect it to change moderately frequently, providing a list of sample values will help users enter the right thing.